samples/.gitignore

```diff
@@ -11,7 +11,6 @@ _ReSharper.*
 *.cache
 *.tlog
 *.log
-*.obj
```

Comment:
Same remark. As we want this `.gitignore` file to be a sample by itself, we shouldn't touch it.